### PR TITLE
Code refactoring

### PR DESCRIPTION
**Pull request by aashishdhawan**

### CODE REVIEW

File: Demo/WhisperDemo/WhisperDemo/ViewController.swift [hunk 1]
```diff
@@ -3,13 +3,13 @@ import Whisper
 
 class ViewController: UIViewController {
 
-  lazy var scrollView: UIScrollView = UIScrollView()
+  lazy var scrollView = UIScrollView()
 
-  lazy var titleLabel: UILabel = {
+  lazy var titleLabel: UILabel = { [unowned self] in
```
Comment (zenangst) [inline Demo/WhisperDemo/WhisperDemo/ViewController.swift]: Why did you add `[unowned self]` here?
`self` is not referenced inside of the closure so I think we can go without it.

File: Demo/WhisperDemo/WhisperDemo/ViewController.swift [hunk 2]
```diff
@@ -58,7 +58,7 @@ class ViewController: UIViewController {
     return button
     }()
 
-  lazy var containerView: UIView = {
+  lazy var containerView: UIView = { [unowned self] in
```
Comment (zenangst) [inline Demo/WhisperDemo/WhisperDemo/ViewController.swift]: Same here, `self` is not referenced in the closure so it holds no reference to it.

**Comment by aashishdhawan** [comment]
@zenangst That was a slip. Reverted that commit. Sorry for inconvenience caused.
